### DISCUSSION TITLE
欢迎你来一起听播客！

### DISCUSSION THREAD

**Discussion by yenche123** [yenche123/podcast-together#1] [公告]
Hey, what's up!
很高兴认识你~ 
希望这个项目能让听播客不再是一件很孤单的事，它还是抬头计划的一部分，有没有注意到我们在移动互联网时代大家都在低头，我搭地铁时，放眼望去几乎所有人都在低头看手机，只有少数听音乐和听播客者大概会抬头了。
一直低头对肩颈很不好，现在，又多了一个选项可以玩手机不低头了！

如果有任何问题，欢迎来 Github 讨论区跟我们交流，想在这里分享播客的也可以哦！
（但分享内容必须包含"一起听"的链接🤣(^///^)

**Comment by icm-ai** [comment]
支持支持，很喜欢听播客。

**Comment by Kugaaa** [comment]
非常感谢大佬，好用~UI 也特别美观
可以请教一下（太菜了完全不懂前端），我看大佬在语雀上粘贴了 Vercel 部署的域名 https://podcast-together-neon.vercel.app/
我就 fork 了项目部署了一下，发现竟然部署成功了（瞎搞），但是输入链接解析的时候会一直解析中，查看日志
![image](https://user-images.githubusercontent.com/99192047/231811053-aa6d80cc-c77f-48ea-a87c-bf219593dacc.png)

猜测原因：
应该是 /undefined/pt-service 接口相关的路径没有配置，看了下部署文档，应该是需要自己部署一套 laf 应用（还不清楚这个 laf 是什么哈），可以使用 lafyun，操作完成、暴露接口后配置到 VITE_API_URL 等相关配置上吗？

十分感谢~

**Reply by yenche123** [reply]
你有配置环境变量吗？如果前端是部署在 Vercel 上的话，需要在它的 Settings - Environment Variables 中进行配置（见下图），这一步对应语雀那篇文档的 [配置 .env.local](https://www.yuque.com/cuiyanzhe/podcast-together/etgegl#Lj5p7)，等号左边作为 Key、等号右边做为 Value 填入Environment Variables 中即可。
<img width="938" alt="image" src="https://user-images.githubusercontent.com/28533125/231818658-7da7e53d-4f0a-4205-9b9d-0c22f03272a5.png">

另外，Laf 作为后端云 Serverless 平台，今年发表了 v1.0 正式版，有许多更动，我还没来得及做适配。你的问题也可能跟此有关。

**Reply by Kugaaa** [reply]
感谢感谢，确实有很多步骤我还没做，我再尝试一下
